packages/editor/src/components/post-title/style.native.scss

```diff
@@ -8,6 +8,6 @@
 }
 
 .dimmed {
-	opacity: 0.2;
+	opacity: 1;
```

Comment:
As I wrote above: what if we move this to  separate PR to keep release process clean and avoid mistakes in release changelog